packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -251,7 +252,25 @@ export class AppManagementClient implements DeveloperPlatformClient {
         cacheExtraKey: userId,
       })
 
-      if (userInfoResult.currentUserAccount) {
+      if (getPartnersToken() && userInfoResult.currentUserAccount) {
+        const organizations = userInfoResult.currentUserAccount.organizations.nodes.map((org) => ({
+          name: org.name,
+        }))
+
+        if (organizations.length > 1) {
+          throw new Error('Multiple organizations found for the CLI token')
```

Comment:
let's report this as a `BugError` since is something that should never happen and we want it properly reported